### ISSUE TITLE
how to

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#486]
**how to**

Hallo. 
I am trying to get only the marine fungi from a specific area.
So, I d like to use these 3 filters: area, fungi, marine
Is there a way to do so? 

Thank you in advance

-----
fbitem-4cd51ab3c35a7b3305c9ec1422d26bc93a21a482
User provided contact info: haris.zafr@gmail.com
System: Firefox 55.0.0 / Ubuntu 0.0.0
Referer: https://www.gbif.org/occurrence/search
Window size: width 1715 - height 872
[API log](http://elk.gbif.org:5601/app/kibana#/discover/UAT-Varnish-403s?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2017-09-25T10:02:43.013Z',mode:absolute,to:'2017-09-25T10:08:43.013Z'))&_a=(columns:!(request,response,clientip),filters:!(),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499%20AND%20(request:%22%2F%2Fapi.gbif.org%22)')),sort:!('@timestamp',desc))&indexPattern=uat-varnish-*&type=histogram)
[Site log](http://elk.gbif.org:5601/app/kibana#/discover/UAT-Varnish-403s?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2017-09-25T10:02:43.013Z',mode:absolute,to:'2017-09-25T10:08:43.013Z'))&_a=(columns:!(request,response,clientip),filters:!(),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E399%20AND%20(request:%22%2F%2Fdemo.gbif.org%22)')),sort:!('@timestamp',desc))&indexPattern=uat-varnish-*&type=histogram)

**Comment by MortenHofft** [comment]
In occurrence search you can [search for fungi and your area of interest](https://www.gbif.org/occurrence/search?has_coordinate=true&has_geospatial_issue=false&taxon_key=5&geometry=POLYGON%20((-4.21875%2051.17934297928927,%20-4.21875%2057.70414723434193,%205.625%2057.70414723434193,%205.625%2051.17934297928927,%20-4.21875%2051.17934297928927))), but not for marine species.
Ideally limiting you search to water areas would do so i guess, but you would need to filter them locally after download to make sure to exclude species that aren't marine. You can also search for [multiple species](https://www.gbif.org/occurrence/search?has_coordinate=true&has_geospatial_issue=false&taxon_key=2612450&taxon_key=9070616&taxon_key=2612452&geometry=POLYGON%20((-4.21875%2051.17934297928927,%20-4.21875%2057.70414723434193,%205.625%2057.70414723434193,%205.625%2051.17934297928927,%20-4.21875%2051.17934297928927))) - if you have a lot, then you will need to use the [API](https://www.gbif.org/developer/occurrence#predicates).

If you do not know which species you are looking for, then dedicated checklists like [WoRMS](https://www.gbif.org/dataset/2d59e5db-57ad-41ff-97d6-11f5fb264527) might be useful. The [API](https://www.gbif.org/developer) also allows for [searching marine species](http://api.gbif.org/v1/species/search?habitat=marine&dataset_key=d7dddbf4-2cf0-4f39-9b2a-bb099caae36c&highertaxon_key=5). Be aware that which species is labeled as marine entirely depends on the published datasets the backbone is build from and might not be complete. Those species could subsequently be used to further filter the downloaded data.

So in summary: You can search for area+fungi but not marine. There is currently no way to simply download occurrences of marine fungi without knowing the species up front. Using the API you can get species in the backbone that are labeled as marine.

Hope that helps
